pom.xml

```diff
@@ -249,6 +249,54 @@
                     <groupId>org.owasp</groupId>
                     <artifactId>dependency-check-maven</artifactId>
                     <version>${dependency.check.maven.plugin.version}</version>
+                    <configuration>
```

Comment:
Do we need failBuildOnCVSS?

Reply:
we do i was trying to figure  out the CVSS threshold